digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/pointasset/OracleObstacleDao.scala

```diff
@@ -91,7 +86,15 @@ object OracleObstacleDao {
         propertyType = row.property.propertyType,
         required = row.property.propertyRequired,
         values = rows.flatMap { assetRow =>
-           Seq(PropertyValue(assetRow.property.propertyValue, Option(assetRow.property.propertyDisplayValue)))
+
+          val isSpecialCase = specialCases.contains(assetRow.property.publicId.toString )
+          val isPropertyValueEmpty = assetRow.property.propertyValue.toString.trim.isEmpty
+
+          val finalValue = if ( isSpecialCase && isPropertyValueEmpty ) "0"
+                           else assetRow.property.propertyValue
+
+          Seq(PropertyValue(finalValue, Option(assetRow.property.propertyDisplayValue)))
```

Comment:
This problem already happens in other places, will be fixed when the task to fix bugs in the batches will be merged to master
Let's try to create a function to avoid creating the same code in multiple places

Reply:
Require more time to investigate and implement a solution.